client/src/components/screens/PatientSubmission/index.jsx

```diff
@@ -372,14 +372,17 @@ function PatientSubmissionScreen(props) {
 
       const ageInDay = moment(new Date()).diff(currentPatient.birthDate, 'days');
 
+      const submittedStatus = submitted ? 'on-hold' : (status || 'draft')
```

Comment:
I think you could incorporate enums

Reply:
ok, will re-use the StatusType from Francis status modal